pkg/cfg/cfgmgr.go

```diff
@@ -163,71 +165,105 @@ func (c * CfgMgr) getRandomPassword() string {
     return string(out)
 }
 
-func (c *CfgMgr) CreateDB(serviceName, userName string) error {
+func (c *CfgMgr) CreateDB(serviceName, userName string) (bool, error) {
+    dbObject, err := c.getDbObject()
+    if err != nil {
+        return false, err
+    }
+    _, err = dbObject.Exec(fmt.Sprintf("SHOW DATABASE LIKE %s", serviceName))
+    if err == nil {
+        zap.L().Info(fmt.Sprintf("Database %s already exists", serviceName))
+        return false, nil
```

Comment:
Isn't DB technically created at this point? Can you return `true, nil` to that effect?

Reply:
false is returned as we have nothing to update for DB in consul.